### PR TITLE
[RW-7652][risk=no] Mitigate egress suspension test timeouts

### PR DESCRIPTION
**Pull request by calbach**
Problem: running the egress notebook is problematic, as we may be suspended in the middle of the run (this pauses the notebook, and makes it unresponsive). Furthermore, we can get dialogs when trying to navigate away from the notebook page, since it may not be fully saved or completed running.

Solution: run the notebook in the "background", and then refresh/check for suspension in another tab. This avoids any dependency on the code running to completion.

### CODE REVIEW

File: e2e/tests/nightly/egress-suspension.spec.ts [hunk 1]
```diff
@@ -22,13 +23,26 @@ describe('egress suspension', () => {
...
-
-    await notebookPage.save();
 
-    console.log('Awaiting security suspension');
-    await notebookPage.waitForSecuritySuspendedStatus(true);
+    // Let the egress notebook run in the background; intentionally do not await here.
+    // It may not complete since our user may have their compute disabled while
+    // this is still running.
+    console.log('Generating egress via notebook');
+    notebookPage.runCodeFile(1, egressFilename, 5 * 60 * 1000).catch((err) => {
```
Comment (calbach) [inline e2e/tests/nightly/egress-suspension.spec.ts:32]: Please hold for  a second - I'm reworking this part

Comment (calbach) [reply e2e/tests/nightly/egress-suspension.spec.ts]: OK, PTAL.

File: e2e/utils/runtime-utils.ts [hunk 2]
```diff
@@ -17,3 +17,35 @@ export async function initializeRuntimeIfModalPresented(page: Page): Promise<voi
...
+    await page.waitForXPath('//*[@data-test-id="security-suspended-msg"]', { visible: true });
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
+export async function waitForSecuritySuspendedStatus(
+  page: Page,
+  suspended: boolean,
```
Comment (aweng98) [inline e2e/utils/runtime-utils.ts:32]: could also make `suspended` optional provide a default value of true.

**Review by aweng98 — APPROVED** [review]
LGTM!

**Pull request merged**